### ISSUE TITLE
[HELP]All accounts getting Rate limit Banned

### ISSUE THREAD

**Issue by AlexX1-dev** [rdeepak2002/reddit-place-script-2022#282]
# Bug Report Form

## Checklist

- [x] I am using the latest version of the code.
- [x] I am absolutely sure I read the README in its entirety.
- [x] I have made sure this is not a duplicate issue.
- [x] I have filled out this form correctly.

## Describe the bug

I have like 44 Accounts and like 10 of them are already rate limit banned, what cab I do to avoid that?

## To Reproduce

Steps to trigger the bug:

1. Run the script
2. Wait
3. See accounts being rate limit banned

## Expected behavior

My Accounts shouldn't be rate limit banned

## Additional context

What can I do?

**Comment by Risewill23** [comment]
for now you can increase the "thread_delay" in the config until the proxies bug is fixed

**Comment by wl1400** [comment]
How are you even able to use this with the new anti-bot patch?

**Comment by tylerval18** [comment]
> for now you can increase the "thread_delay" in the config until the proxies bug is fixed

What delay do you recommend to use?

**Comment by Risewill23** [comment]
Matters on how much accounts do you have

**Comment by AlexX1-dev** [comment]
For like 40 Accounts what should I use?

**Comment by AlexX1-dev** [comment]
> How are you even able to use this with the new anti-bot patch?

What patch?

**Comment by Risewill23** [comment]
60 seconds would likely be the best but I hope the proxies will be fixed soon

**Comment by AlexX1-dev** [comment]
All my 44 Bots are banned..... RIP

**Comment by wl1400** [comment]
> > How are you even able to use this with the new anti-bot patch?
> 
> What patch?

Whenever you try to place a pixel with multiple accounts on the same IP you get a 15 year timeout on the account, may I ask, what method did you use to get your accounts?

**Comment by Risewill23** [comment]
> > > How are you even able to use this with the new anti-bot patch?
> > 
> > 
> > What patch?
> 
> Whenever you try to place a pixel with multiple accounts on the same IP you get a 15 year timeout on the account, may I ask, what method did you use to get your accounts?

thats why I'm trying to fix the proxy bug

**Comment by AlexX1-dev** [comment]
> Whenever you try to place a pixel with multiple accounts on the same IP you get a 15 year timeout on the account, may I ask, what method did you use to get your accounts?
Creating them using a script with verified email

**Comment by wl1400** [comment]
> > > > How are you even able to use this with the new anti-bot patch?
> > > 
> > > 
> > > What patch?
> > 
> > 
> > Whenever you try to place a pixel with multiple accounts on the same IP you get a 15 year timeout on the account, may I ask, what method did you use to get your accounts?
> 
> thats why I'm trying to fix the proxy bug

Ah I see, appreciate your efforts.

**Comment by wl1400** [comment]
> > Whenever you try to place a pixel with multiple accounts on the same IP you get a 15 year timeout on the account, may I ask, what method did you use to get your accounts?
> 
> Creating them using a script with verified email

Are you using [ItCameFr0mMars](https://github.com/ItCameFr0mMars) account gen?

**Comment by AlexX1-dev** [comment]
> > > Whenever you try to place a pixel with multiple accounts on the same IP you get a 15 year timeout on the account, may I ask, what method did you use to get your accounts?
> > 
> > 
> > Creating them using a script with verified email
> 
> Are you using [ItCameFr0mMars](https://github.com/ItCameFr0mMars) account gen?

No, I use a modified version of the original version I changed some things by myself

**Comment by wl1400** [comment]
> > > > Whenever you try to place a pixel with multiple accounts on the same IP you get a 15 year timeout on the account, may I ask, what method did you use to get your accounts?
> > > 
> > > 
> > > Creating them using a script with verified email
> > 
> > 
> > Are you using [ItCameFr0mMars](https://github.com/ItCameFr0mMars) account gen?
> 
> No, I use a modified version of the original version I changed some things by myself

Ah ok, thanks.

**Comment by jdroo** [comment]
Maybe randomize the delay?

**Comment by tylerval18** [comment]
Most of the accounts I made with the account gen got banned, but most of the accounts I made manually didn't, could this be related somehow?

**Comment by AlexX1-dev** [comment]
> Most of the accounts I made with the account gen got banned, but most of the accounts I made manually didn't, could this be related somehow?

hmm maybe you used different mails?

**Comment by AlexX1-dev** [comment]
> > > > How are you even able to use this with the new anti-bot patch?
> > > 
> > > 
> > > What patch?
> > 
> > 
> > Whenever you try to place a pixel with multiple accounts on the same IP you get a 15 year timeout on the account, may I ask, what method did you use to get your accounts?
> 
> thats why I'm trying to fix the proxy bug

Do I need a proxy for each account?

**Comment by wl1400** [comment]
> > > > > How are you even able to use this with the new anti-bot patch?
> > > > 
> > > > 
> > > > What patch?
> > > 
> > > 
> > > Whenever you try to place a pixel with multiple accounts on the same IP you get a 15 year timeout on the account, may I ask, what method did you use to get your accounts?
> > 
> > 
> > thats why I'm trying to fix the proxy bug
> 
> Do I need a proxy for each account?

Yes

**Comment by Lr3ker** [comment]
Hate being the noob here but don't have much experience with proxies, do either of you have a link/instructions on how to set proxies up for this
